docs/src/snap/howto/image-management.md

```diff
@@ -0,0 +1,66 @@
+# How to manage images
```

Comment:
Does this page need to be listed in the `index.md`?

Reply:
+1